apps/api/src/.env.development

```diff
@@ -25,7 +25,7 @@ REDIS_CACHE_FAMILY=
 REDIS_CACHE_KEY_PREFIX=
 REDIS_CACHE_ENABLE_AUTOPIPELINING=true
 
-IS_REQUEST_RATE_LIMITING_ENABLED=false
+IS_API_RATE_LIMITING_ENABLED=false
```

Comment:
"API Rate Limiting" aligns naming with the use-cases under development. If we decide to add Web Socket rate limiting or other types, we can then toggle these features independently.